controllers/nstemplateset/namespaces.go

```diff
@@ -154,9 +157,18 @@ func (r *namespacesManager) ensureInnerNamespaceResources(logger logr.Logger, ns
 	if err := r.Client.Update(context.TODO(), namespace); err != nil {
 		return r.wrapErrorWithStatusUpdate(logger, nsTmplSet, r.setStatusNamespaceProvisionFailed, err, "failed to update namespace '%s'", nsName)
 	}
+	// ensure the namespace created is ready
+	userNamespaceCreated := &corev1.Namespace{}
+	if err := r.Client.Get(context.TODO(), types.NamespacedName{Name: namespace.Name}, userNamespaceCreated); err != nil {
+		if errors.IsNotFound(err) {
+			return fmt.Errorf(`Unable to get the usernamespace '%s'`, namespace.Name)
+		}
+	}
+	if userNamespaceCreated.Status.Phase != "Active" {
+		return fmt.Errorf(`Namespace %s is in %s Phase`, userNamespaceCreated.Name, userNamespaceCreated.Status.Phase)
+	}
```

Comment:
why do we need this? We already check if the namespace is active or not
https://github.com/codeready-toolchain/member-operator/blob/218698f00f6cb2d41274bd15100677e651be92e5/controllers/nstemplateset/namespaces.go#L301
if it's not active, then we cannot apply the inner-namespace objects